packages/@tinacms/toolkit/src/packages/react-sidebar/components/Nav.tsx

```diff
@@ -129,9 +129,13 @@ export const Nav = ({
                             sidebarWidth: null,
                             resizingSidebar: false,
                           })
-                          setEdit(false)
-                          if (cms?.api?.tina?.onLogout) {
-                            cms.api.tina.onLogout()
+                          if (cms?.api?.tina?.logout) {
+                            cms.api.tina.logout().then(async () => {
```

Comment:
should the way this is called in LogoutPage be consistent (await vs then())

Reply:
yup, makes sense! I will fix it.

Reply:
Fixed